### DISCUSSION TITLE
How to load existing e-reader levels

### DISCUSSION THREAD

**Discussion by mariomadproductions** [city41/smaghetti#76] [General]
- Convert the .raw e-card dump to .bin using [NEDC_Enc](https://github.com/city41/smaghetti/files/6618451/nedcenc.zip) by CaitSith2
- Remove the data before the "ASR0" string in the .bin file
- Use [sma4comp](https://github.com/LuigiBlood/sma4tool/releases) to decompress the edited .bin file
- Now you have a file you can load into the hex tree mode

**Comment by city41** [comment]
awesome! I was poking at this myself but I didn't get too far. Thanks for figuring that out. I'll add this to the wiki.

**Reply by city41** [reply]
actually, doh, the linked websites have the files. We can't link to those sites here, it's important to not have this repo linked to any copyrighted info. I'm glad you got it working, but I need to remove the link.

**Reply by mariomadproductions** [reply]
Okay. I've attached the tool to the post and just put the author's name instead of a link.
